{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "1a2c8fc1_eca60547",
        "filename": "sc/qa/uitest/calc_tests9/tdf115933.py",
        "patchSetId": 3
      },
      "lineNbr": 29,
      "author": {
        "id": 1000179
      },
      "writtenOn": "2021-09-10T09:02:07Z",
      "side": 1,
      "message": "Why do you need this try/except part ?",
      "revId": "276e34f14c824e17683c8ff734d575cee50166ed",
      "serverId": "6622c302-437a-49ad-a91d-55ba7dcf1615"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "d7632ad5_d7441681",
        "filename": "sc/qa/uitest/calc_tests9/tdf115933.py",
        "patchSetId": 3
      },
      "lineNbr": 29,
      "author": {
        "id": 1001967
      },
      "writtenOn": "2021-09-10T09:56:51Z",
      "side": 1,
      "message": "An error message is displayed, if Calc doesn\u0027t accept the password: The password is incorrect. The file cannot be modified.\n(click OK, and then click Cancel in password dialog)",
      "parentUuid": "1a2c8fc1_eca60547",
      "revId": "276e34f14c824e17683c8ff734d575cee50166ed",
      "serverId": "6622c302-437a-49ad-a91d-55ba7dcf1615"
    }
  ]
}